traffic_ops/app/lib/API/Profile.pm

```diff
@@ -333,6 +345,13 @@ sub update {
 		}
 	}
 
+	if ( defined( $params->{routingDisabled} ) && $params->{routingDisabled} != (0 || 1) ) {
+		print STDERR Dumper($params->{routingDisabled});
+		return $self->alert("Routing disabled is a boolean value.");
+	}
+
+	my $routing_disabled = defined($params->{routingDisabled}) ? $params->{routingDisabled} : 0;
```

Comment:
this won't work. if you send thru routingDisabled: true or routingDisabled: false it will always put a 1 in the database. you might need to consult @dangogh (our resident perl expert) on what the best approach is here. 

Also, you might want to retest thru the API, i keep getting this error when i send thru routingDisabled: false

Routing disabled is a boolean value.

Reply:
Also, you need to add some unit tests to test this new column. at a minimum, create and update a profile thru the api with routingDisabled=true and routingDisabled=false

Reply:
Also, can you add your new field to this api endpoint?

https://github.com/apache/incubator-trafficcontrol/blob/master/traffic_ops/app/lib/API/Profile.pm#L87

Reply:
Also, I created a PR for you that adds this field to the Traffic Portal

https://github.com/dg4prez/incubator-trafficcontrol/pull/4

Reply:
All fixed.